app/config/locale/templates/email-webhook-failed.tpl

```diff
@@ -14,7 +14,7 @@
 <table border="0" cellspacing="0" cellpadding="0" style="padding-top: 10px; padding-bottom: 10px; margin-top: 32px">
     <tr>
         <td style="border-radius: 8px; display: block; width: 100%;">
-            <a class="mobile-full-width" rel="noopener" target="_blank" href="{{redirect}}" style="font-size: 14px; font-family: Inter; color: #ffffff; text-decoration: none; background-color: #FD366E; border-radius: 8px; padding: 9px 14px; border: 1px solid #FD366E; display: inline-block; text-align:center; box-sizing: border-box;">Webhook settings</a>
+            <a class="mobile-full-width" rel="noopener" target="_blank" href="{{protocol}}://{{hostname}}{{redirect}}" style="font-size: 14px; font-family: Inter; color: #ffffff; text-decoration: none; background-color: #FD366E; border-radius: 8px; padding: 9px 14px; border: 1px solid #FD366E; display: inline-block; text-align:center; box-sizing: border-box;">Webhook settings</a>
```

Comment:
Hmm, should we instead have just 1 param, `{{host}}`, which would be `https://instance.com`?

Reply:
Reading such placeholder would be easier than protocol and hostname separated.